taxonomies/ingredients_processing.txt

```diff
@@ -1645,6 +1684,120 @@ wikidata:en: Q7601513
+th: ข้น
+tl: na kondensada
+tr: Kondense
+tt: Куертылган
+uk: згущене
+vi: đặc
+zh: 煉
+
+# description:en:LOW-FAT MILK is a 1% fat milk, sold in the United States
+# description:en:Skimmed milk (British English) is made when all the cream is removed from whole milk. It tends to contain around 0.1% fat. Láit écrémé has a maximum of 0.5% fat.
+# description:en:Skim milk (American English), is made when all the cream is removed from whole milk. It tends to contain around 0% fat.
+# Nonfat, skim milk and skimmed milk are often used as synonyms and the
+# regulatory differences are small, with the average being between 0.0% and
+# 0.2% fat. We are thus using only one taxonomy entry for all milks with
+# a maximum of 0.5% fat
+
+en: skimmed, skim, non-fat, nonfat, fat-free, low-fat, lowfat, defatted, 1% fat, with 0.5% milk fat
+ar: منزوع الدسم
+bg: обезмаслено, обезмаслно, обезмаслен, обезмаслена, обезмаслени
+ca: desnatada, descremada, desgreixat, desgreixada, desgreixats, desgreixades, sense greix, baixa en greixos
+cs: odstředěné, odstředěné
+da: skummet
+de: Mager, entrahmtes, entrahmte, entfettet, entfettete, entfetteter, entfettetes, entfetteten, entfettetem, fettarme
+el: αποβουτυρωμένο
+es: desnatada, descremada, desengrasado, desengrasada, desgrasado, desgrasada, sin grasas, baja en grasa
+et: koorimata, kooritud
+fa: خشک بدون چربی
+fi: rasvaton, rasvatonta, Vähärasvainen
+fr: écrémé, maigre, dégraissé, dégraissée, dégraissés, dégraissées
+gl: desnatado
+he: רזה
+hr: obrano, obrane, obrani
+hu: sovány, zsírtalanított, fölözött, alacsony zsírtartalmú
+id: skim
+is: undanrennur
+it: scremato, magro, sgrassato, sgrassata, senza grassi, magro
+ja: 脱脂
+lt: nugriebtas, nugriebto, lieso
+lv: vaj, vāj
+mk: обрано
+ms: skim
+nb: skummet
+nl: mager, magere, ontvet, ontvette, tapte, onder, afgeroomde, vetvrije
+no: skummet
+pl: odtłuszczone, chude, o obniżonej zawartości tłuszczu, odtłuszczony, odtłuszczona, odtłuszczonego
+pt: magro, desnatado, desengordurado, desengordurada
+ro: degresat
+ru: обезжиренноё, и обезжиренноё, обезжиренный, обезжиренное, безжиренное, и обезжиренное
+sk: odstredené
+sl: posneto
+sq: i skremuar
+sv: skum, lätt
+th: ขาดมันเนย
+tr: yağsız, kaymağı alınmış
+uk: Знежирене
+vi: gầy
+zh: 脱脂
+
+< en:skimmed
+en: semi-skimmed, partially defatted, reduced-fat, partly skimmed, partially skimmed, part skim
+bg: Полуобезмаслено, частично обезмаслен, частично обезмаслено
+ca: parcialment desnatada, semidesnatada, parcialment desgreixat, parcialment desgreixada, parcialment desgreixats, parcialment desgreixades
+de: teilentrahmte, teilentfettet, teilentfettete, teilentfettetes, teilentfetteter, teilentfetteten, teilentfettetem, halbentrahmte
+es: parcialmente desnatada, semidesnatada, semidescremada, parcialmente descremada
+fi: kevyt
+fr: semi-écrémé, demi-écrémé, partiellement écrémé
+hr: poluobrano, djelomično obrano, djelimično obrano
+hu: zsírszegény, félzsíros
+is: létt
+it: semi-magro, parzialmente scremato, semiscremato
+nb: lett
```

Comment:
I don’t know Icelandic _or_ Norwegian, but again, with their proximity to Danish and Swedish, I would assume that `l[eé]tt` also means “light” similar to Danish “let” and Swedish “lätt”. “Light” can refer to not just a reduction of fat content, but to any reduction of calories.
```suggestion
it: semi-magro, parzialmente scremato, semiscremato
```